app/src/main/java/com/example/playlistmaker/models/Track.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.playlistmaker.models
+
+data class Track(
+    val trackName: String, // Название композиции
+    val artistName: String, // Имя исполнителя
+    val trackTime: String,// Продолжительность трека
+    val artworkUrl100: String // Ссылка на изображение обложки
```

Comment:
```suggestion
    val artworkUrl100: String, // Ссылка на изображение обложки
```
Такие запятые можно ставить, когда описываешь некоторое кол-во элементов (пр. аргументы функции или конструктора). Это называется trailing comma в котлине, прочитать про её можешь ссылке - https://kotlinlang.org/docs/coding-conventions.html#trailing-commas